Remotion/Web/Core/ExecutionEngine/Infrastructure/NoneTransactionStrategy.cs

```diff
@@ -76,7 +76,7 @@ public override void EnsureCompatibility (IEnumerable objects)
       OuterTransactionStrategy.EnsureCompatibility (objects);
     }
 
```

Comment:
I don't really understand the reasoning. Besides, this is another case where the nullability of the property in the base class is only forced by the implemented null object.

Reply:
Good point. So, acutally the NullTransactionStrategy.Outer should also return NullTransactionStrategy instead of null. then we can make this non-nullable. Need to check that there's no issues with retunring the same reference, etc, but this would be the correct bugfix.

Reply:
probably hsould be a separate ticket. it's a change that goes beyond just guarding the nullability. before or after, as you like.